<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>startFrom</class>
 <widget class="QWidget" name="startFrom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文新魏</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>第一关</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/resources/level.png</normaloff>:/resources/level.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文新魏</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>第二关</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/resources/level.png</normaloff>:/resources/level.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>140</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文新魏</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>第三关</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/resources/level.png</normaloff>:/resources/level.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文新魏</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>导入地图</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/resources/opeicon.png</normaloff>:/resources/opeicon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>260</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文新魏</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>寻访</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/resources/cardstone.png</normaloff>:/resources/cardstone.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
